pom.xml

```diff
@@ -1038,7 +1038,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
 	<configuration>
-          <parallel>classes</parallel>
+          <!--<parallel>classes</parallel>-->
```

Comment:
I introduced this during hive 2.3.3 migration effort in order to quicken the unit tests execution but this is leading to failures during 'all' mode because of the singleton instance of minikdc and  some other classes which causes the tests not to run successfully.